hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/DFSPathSelector.java

```diff
@@ -121,28 +121,30 @@ public static DFSPathSelector createSourceSelector(TypedProperties props,
       eligibleFiles.sort(Comparator.comparingLong(FileStatus::getModificationTime));
```

Comment:
don't we need any fix in listEligibleFiles()? this method filters files based on mod time > checkpoint time. I thought fix is to make this mod time >**=** checkpoint time.

Reply:
@nsivabalan I've explain some implications of this in #2850